src/lib/description-helpers/endpoint-parsers.test.ts

```diff
@@ -0,0 +1,65 @@
+import { it, expect, describe } from "vitest";
+import {  getParameterPaths, getExample } from "./endpoint-parsers"; //getResponseParameterExample,
+import { endpoint } from "../__fixtures__/endpoint";
+
+describe('getParameterPaths', () => {
+  it('should return the correct parameter paths and example paths for the endpoint', () => {
+    const expectedPaths = [
+        "POST|request|application/json|body|properties|products|items|properties|productId",
+        "POST|request|application/json|body|properties|products|items|properties|quantity",
+        'POST|request|application/json|body|properties|shippingAddress|properties|street',
+        'POST|request|application/json|body|properties|shippingAddress|properties|city',
+        'POST|request|application/json|body|properties|shippingAddress|properties|country',
+        'POST|response|200|application/json|body|properties|id',
+        'POST|response|200|application/json|body|properties|status',
+        'POST|response|200|application/json|body|properties|items|items|properties|products|items|properties|productId',
+        'POST|response|200|application/json|body|properties|items|items|properties|products|items|properties|quantity',
+        'POST|response|200|application/json|body|properties|items|items|properties|products|items|properties|price',
```

Comment:
![image](https://github.com/Superflows-AI/openapi-llm/assets/33871096/153be56a-afb4-4135-9b39-f70dad41dd91)

Great you added some tests! :smile: